i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
The scripts are currently not "managed" by maven.
We would need to integrate it first.